.werft/observability/monitoring-satellite.ts

```diff
@@ -191,3 +183,29 @@ function ingressReady(namespace: string, name: string): boolean {
     werft.log(sliceName, `${name} ingress not ready.`)
     return false
 }
+
+/**
+ * installSetup creates or replaces everything that is needed to run monitoring-satellite.
+ * This setup includes Custom Resource Definitions(CRDs), kubernetes operators and namespaces.
+ */
+function installSetup() {
+    // Adds a label to the namespace metadata.
+    // This label is used by ServiceMonitor's namespaceSelector, so Prometheus
+    // only scrape metrics from its own namespace.
+    exec('kubectl apply -f observability/monitoring-satellite/manifests/namespace.yaml', {silent: true})
+    exec('kubectl apply -f observability/monitoring-satellite/manifests/podsecuritypolicy-restricted.yaml', {silent: true})
+
+    let crdExists = exec('kubectl get customresourcedefinitions.apiextensions.k8s.io alertmanagers.monitoring.coreos.com', {silent: true}).code == 0
+    if (crdExists) {
+        werft.log(sliceName, "CRDs already exists, replacing CRDs'")
+        exec('for yaml in $(find observability/monitoring-satellite/manifests/prometheus-operator/ -type f -name "*CustomResourceDefinition.yaml"); do cat $yaml | kubectl delete -f -; done', {slice: sliceName})
+        exec('for yaml in $(find observability/monitoring-satellite/manifests/prometheus-operator/ -type f -name "*CustomResourceDefinition.yaml"); do cat $yaml | kubectl create -f -; done', {slice: sliceName})
+    } else {
+        werft.log(sliceName, "CRDs don't exists, creating from scratch")
+        exec('find observability/monitoring-satellite/manifests/prometheus-operator/ -type f -name "*CustomResourceDefinition.yaml" | kubectl create -f -', {slice: sliceName})
```

Comment:
This exec is different from the one in the `if` branch. Do they install different things? If not then it might be easier to simplify it so that it goes like this 

**If CRDs have to be installed separately from the rest**

```ts
if (crdExists) {
  // Delete CRDs
}
// Install CRDs
// Install non-CRDs
```

**If CRDs don't have to be installed separately from the rest**

```ts
if (crdExists) {
  // Delete CRDs
}
// Install prometheus operator
```

That way you don't need to install CRDs in both the `if` and `else` branch :)

Reply:
CRDs do need to be created first! If Prometheus-operator is created without the CRDs installed then it remains in crashloop until someone manually restarts it

Reply:
And thanks for the reminder of excluding unnecessary `else` 🙂

Reply:
@ArthurSens Thanks for clarifying why we need to install CRDS first 🤗